.github/workflows/ids-check.yml

```
@@ -0,0 +1,76 @@
+name: ids-check
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: ids-check
+    runs-on: ubuntu-24.04
+    strategy:
+      fail-fast: false
```

Why do we not want to fail fast?

---

This just came from my default template for workflows. Removed.